conf/defaults.yaml

```diff
@@ -226,7 +226,7 @@ topology.eventlogger.executors: null
 topology.tasks: null
 # maximum amount of time a message has to complete before it's considered failed
 topology.message.timeout.secs: 30
-topology.multilang.serializer: "org.apache.storm.multilang.JsonSerializer"
+topology.multilang.serializer: "org.apache.storm.multilang.MessagePackSerializer"
```

Comment:
While having the `MessagePackSeriailzer` built-in would be amazing, changing it to the default serializer would break a lot of multi-lang libraries, so I don't think that's a great idea.  Let people opt in.